### DISCUSSION TITLE
Run into a fatal error while trying to update the submodules

### DISCUSSION THREAD

**Discussion by Zoe096423** [FunkinCrew/Funkin#3050] [Compiling help] [answered]
I was following the building guild in order to download the source code, and when I'm on this step:
`git submodule update --init --recursive`
The following error popped out:
`fatal: could not get a repository handle for submodule 'art'`

I've researched a bit, and the codes such as `git submodule deinit art` or `git init` did not solve anything.
Does anyone have clues about these errors and how I can solve them?

**Comment by NotHyper-474** [comment, marked as answer]
Did you actually run `git init` or is that just short for `git submodule init art`?

If you can see the folders for the submodules try deleting them and running `git submodule update --init --recursive` again

**Reply by Zoe096423** [reply]
I tried both `git init` and `git submodule init art`.
I'll try the method below.

**Reply by Zoe096423** [reply]
Oh, I think it worked! Thank you for the advice!
